v_3_2_4/worker_template.go

```diff
@@ -200,8 +200,7 @@ coreos:
     command: stop
     mask: true
   - name: systemd-networkd-wait-online.service
-    enable: true
-    command: start
+    mask: true
```

Comment:
Doesn't it mean it will be basically disabled?
What does this unit do?

Reply:
this unit checks that all interfaces in `networkctl` are in status `configured` in some random cases, when interface comes up w/o ipv6 link-local interface, it will be in `configuring` state and this systemd service will fail after timeout.

See here: https://github.com/giantswarm/giantswarm/issues/2553
I've also created an upstream bug: https://github.com/systemd/systemd/issues/8524